cura/API/Account.py

```diff
@@ -114,14 +118,15 @@ def setSyncState(self, service_name: str, state: int) -> None:
             self._sync_state = SyncState.SYNCING
         elif any(val == SyncState.ERROR for val in self._sync_services.values()):
             self._sync_state = SyncState.ERROR
+            self._setManualSyncEnabled(True)
         else:
             self._sync_state = SyncState.SUCCESS
 
         if self._sync_state != prev_state:
             self.syncStateChanged.emit(self._sync_state)
 
             if self._sync_state == SyncState.SUCCESS:
-                self._last_sync_str = datetime.now().strftime("%d/%m/%Y %H:%M")
+                self._last_sync_str = datetime.now().strftime("%m/%d/%Y %H:%M")
```

Comment:
Uh? Why? Only America uses this. I think it's way better to have it Day/month/year. As far as I know, this is also how we handle other places where we show dates (eg; "your print is completed by" in the cloud / connect)

Reply:
As per design: https://jira.ultimaker.com/browse/CURA-7290?focusedCommentId=103361&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-103361